lib/Controller/ColumnController.php

```diff
@@ -92,7 +105,10 @@ public function create(
 			$selectionOptions,
 			$selectionDefault,
 
-			$datetimeDefault) {
+			$datetimeDefault,
+
+			$viewId,
+			$selectedViewIds) {
```

Comment:
Should this be typed? 
```suggestion
			?array $selectedViewIds) {
```